### DISCUSSION TITLE
Open Video from remote URL

### DISCUSSION THREAD

**Discussion by chris-der-kleine** [SubtitleEdit/subtitleedit#4930] [Q&A] [answered]
VLC by itself allows to open network streams.

Having the ability to open a network stream could be useful to subtitle youtube videos.

**Comment by OmrSi** [comment, marked as answer]
Right now, you can do that using MPV and youtube-dl.
Download youtube-dl https://youtube-dl.org/downloads/latest/youtube-dl.exe then place the file in the SE's data file `%appdata/Subtitle Edit`
After that, go to video player settings and set MPV as your video player.
Then you can see an option in menu `Video -> Open video from URL…`

**Reply by chris-der-kleine** [reply]
Oh. I didn't knew that! Thank you!

Is there also a way in which I can also get waveforms without having to download the audio from youtube?

**Reply by OmrSi** [reply]
Sadly, no.
SE needs access to the full video to generate the waveform.
But in this case, MPV is handling playing the video, so it's not stored locally, which means a wave can't be generated.

Although, you can download the video using `youtube-dl`, just open a CMD in the folder and write `youtube-dl.exe URL`
